src/MHTimer.js

```diff
@@ -1422,21 +1434,25 @@ function addRemind(timerRequest, message) {
         responses.unshift('Hi there! Reminders are only sent via PM, and I\'m just making sure I can PM you.');
 
     // Send notice of the update via PM.
-    message.author.send(responses.join(' ')).catch(() =>
-        Logger.error(`Reminders: notification failure for ${message.author.username}.`),
-    );
+    await message.author.send(responses.join(' ')).catch(() => {
```

Comment:
When using `await`, do not also use `Promise#then` or `Promise#catch`. Instead, wrap the awaited call with a try - catch block.